Presentation.Web/Content/less/kitos.less

```diff
@@ -1538,4 +1538,13 @@ tbody.bordered > tr > td {
 }
 .boldParagraph{
     font-weight:bold;
+}
+
+.longTextField{
+    width: 300px;
+}
+
+.referenceLinkField {
```

Comment:
hmm.... måske lige groft nok for forskellige views har forskellige behov for bredder... måske skal det scopes lidt mere så ændringer i relationer ikke rippler på interfaces og omvendt.